mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown

```diff
@@ -312,6 +312,11 @@ The following arguments are supported:
   Google Compute Engine Public IPs and Google Prod IPs. Structure is
   [documented below](#nested_master_authorized_networks_config).
 
+* `data_retention_config` -
```

Comment:
It looks like this is in the Cloud Composer v1 section - is that correct? The version of it in the v2 section says it's only supported on 2.0.23 or higher.

Reply:
Removed.